rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2191,6 +2195,50 @@ void clearRetiredFederation_whenHavingOldFederation_removesOldFederation() {
         assertThat(oldFederation, is(nullValue()));
     }
 
+    @Test
+    @Tag("create and process svp fund transaction unsigned")
+    void createAndProcessSvpFundTransactionWithoutSignatures_savesExpectedTransactionInStorage() throws InsufficientMoneyException {
```

Comment:
Any unexpected corner case we'd like to test?

No UTXOs available at the moment? Fees too high? InsufficientMoneyException?

Reply:
mmm, I think that should be a test for `completeTx` method. Wdyt?

Reply:
i mean, if there are no utxos, `completeTx` should throw the exception. Is not being thrown from these new methods.